bootstrap/bootstrap/bootstrap.py

```diff
@@ -86,12 +93,14 @@ def pull_core(self) -> None:
         lines: int = 0
         # map each id to a line
         id_line: Dict[str, int] = {}
-        for line in self.low_level_api.pull(f"{image}:{tag}", stream=True, decode=True):
-            if len(line.keys()) == 1:
-                # in some cases there is only "status", print that on the last line
-                screen.addstr(lines, 0, line["status"])
-                continue
-            try:
+        try:
```

Comment:
why is there a try without exception and with a finally when a for is inside try ?

Reply:
The exception is raised by `low_level_api.pull()`. The try-finally blocks makes sure that the curses interface is properly finalized before the exception is raised. I intend to re-do that eventually #71 , but that is gonna take more time than I'd like to allocate to it right now